UI/src/stories/molecules/RipaStopDate.stories.js

```diff
@@ -0,0 +1,18 @@
+import RipaStopDate from '@/components/molecules/RipaStopDate'
+
+export default {
+  title: 'Molecules/RipaStopDate',
+  component: RipaStopDate,
+  parameters: {},
+}
+
+export const basic = () => ({
+  components: { RipaStopDate },
+  data() {
+    return {
+      stopDate
+    }
+  },
+  template:
+    '<ripa-stop-date v-model="stopDate"></ripa-stop-date>'
```

Comment:
I would recommend wrapping the component in a div and rendering out the {{ stopDate }} object. This will allow you to test the response in Storybook as you make changes to the form.